src/PrestaShopBundle/Form/Admin/Improve/International/Tax/TaxOptionsType.php

```diff
@@ -55,15 +56,22 @@ class TaxOptionsType extends AbstractType
     /**
      * TaxOptionsType constructor.
      *
+     * Backwards compatibility break introduced in 1.7.8.0 due to extension of TranslatorAwareType
+     *
+     * @param TranslatorInterface $translator
+     * @param array $locales
      * @param bool $ecoTaxEnabled
      * @param FormChoiceProviderInterface $taxAddressTypeChoiceProvider
      * @param FormChoiceProviderInterface $taxRuleGroupChoiceProvider
      */
     public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
```

Comment:
It's a BC :)

Reply:
Sorry, I didn't notice the description hasn't been changed, but the label was added wink